### ISSUE TITLE
Passive/network location service not working after update

### ISSUE THREAD

**Issue by whowantsmybigdata** [GrapheneOS/os-issue-tracker#2692]
Unfortunately passive location service via network completely stopped working with build UP1A.231105.003.2023110700 on P7. 
Before it was always working for me and I really need this functionality a lot. 
No settings changed on my side. 
Supl und psds are both set to graphene-proxy. Wifi and bt search are both switched off.
Will try to attach a logcat later if needed.
Thanks for your help!

**Comment by muhomorr** [comment]
> passive location service via network completely stopped working with build UP1A.231105.003.2023110700 on P7.

How did you test it? I've tested passive location provider and emulated `network` location provider (which is the same as `gps` on GrapheneOS), both work.

**Comment by whowantsmybigdata** [comment]
> How did you test it? 
sorry forgot to mention: I used satstat (because you can differentiate between passive, gps and fused) which was always working when only using "passive" and I realized the issue using osmand or transportr indoors which now both only show the last registered gps location instead of my actual one as it has been before

> emulated `network` location provider (which is the same as `gps` on GrapheneOS)
I have no idea what you mean by that

> both work.
P7 on same build?

**Comment by muhomorr** [comment]
`passive` location provider will provide current location only if GNSS location provider is used by some other app.

Some other app which constantly polled for GNSS-based location must have been running on your device before.

**Comment by whowantsmybigdata** [comment]
But that doesn't sound like it's a new thing and (at least for me) passive location had always been working before with GOS and I didn't change any background services whatsoever?!

so you mean there has to be an service constantly running in the background actively polling gps location?!
1. that sounds like a battery nightmare? 
2. If so, I don't understand the need for passive location anymore, as the last active one would be always correct as well
3. which service could one use to do that more or less efficiently?

Thank you

**Comment by muhomorr** [comment]
Something must have changed in your setup.

For more info about passive location provider, see https://developer.android.com/reference/android/location/LocationManager#PASSIVE_PROVIDER

**Comment by whowantsmybigdata** [comment]
Like I said nothing changed on my side.
Are there any battery optimization settings which need to be disabled in order for the passive location provider to work?

**Comment by thestinger** [comment]
@whowantsmybigdata Is this still an issue for you?

**Comment by whowantsmybigdata** [comment]
yes its still not working, was always updating (automatically) to the newest OS-versions...

**Comment by whowantsmybigdata** [comment]
what can I do to log this to maybe find a solution some day?

**Comment by whowantsmybigdata** [comment]
its really annoying mostly as it was working before and I work as a courier and need to have location data indoors

**Comment by whowantsmybigdata** [comment]
It is still not working for me!

**Comment by whowantsmybigdata** [comment]
I was on the one before. Now with the 2025031400 build it finally works!! thanks for your hard work. Appreciate it so much!
